05-Shared_computing_etiquette.Rmd

```diff
@@ -91,37 +91,44 @@ When you submit jobs, make sure you follow the following guidelines. Again consi
  
  - Think about memory allocation and efficiency.
  
-Consider how much RAM and storage is available for people on the shared computing resource. Try not to overload the resource with a really intensive job or jobs that will use most of the resources and either slow down the efficiency of the work for others or not allow them to perform their work at all.
+Consider how much RAM and storage is available for people on the shared computing resource. Try not to overload the resource with a very intensive job! Jobs that use most of the resources may either slow down the efficiency of the work for others or not allow them to perform their work at all.
 
-This involves:
+Specifically, the etiquette regarding memory allocation includes:
 
    * Not using too many nodes if you don't need to
    * Not using too much RAM on a given node or overall if you don't need to
    * Not submitting too many jobs at once
    * Communicating with others to give them advanced warning if you are going to submit large or intensive jobs
    
-If you have a really large job that you need to perform, talk with the managers of the resource so that you can work out a time when perhaps fewer users would be inconvenienced. Consult the guidelines for your particular resource about how one let's people know about large jobs before you email the administrators of the resource directly. Often their are communications systems in place for users to let each other know about large jobs.
+If you have a really large job that you need to perform, talk with the managers of the resource so that you can work out a time when perhaps fewer users would be inconvenienced. Consult the guidelines for your particular resource about how one lets people know about large jobs before you email the administrators of the resource directly. Often there are communications systems in place for users to let each other know about large jobs.
+
+The illustration below depicts how timing can affect the user experience in using a shared resource. If many people are using the same resource at the same time, especially using up the resources with a heavy task, it might slow down other users, or hinder them from performing their jobs fully. It might be a good idea to target a timeframe when you know might be less crowded.
```

Comment:
```suggestion
The illustration below depicts how timing can affect the user experience in using a shared resource. If many people are using the same resource at the same time, especially using up the resources with a heavy task, it might slow down other users, or hinder them from performing their jobs fully. It might be a good idea to target a timeframe when you know the resource might be less crowded.
```